src/Symfony/Component/Messenger/Transport/Receiver/ReceiverInterface.php

```diff
@@ -58,4 +58,9 @@ public function ack(Envelope $envelope): void;
      * @throws TransportException If there is an issue communicating with the transport
      */
     public function reject(Envelope $envelope): void;
+
+    /**
+     * Remove all messages waiting to be received.
+     */
+    public function purge(): void;
```

Comment:
Instead of having this, can't we reject all messages one by one? I'm not sure what this really means, it seems pretty dramatic (including the fact that it will not `purge` a pre-defined list of things but whatever is in... right now, which might be different than what you just saw on your CLI) 🤔